### PR TITLE
Initial refactor and cleanup of wwlog

### PR DESCRIPTION
**Pull request by gmkurtzer**
Rework `wwlog()` to use `fmt` instead of `log`.

### CODE REVIEW

File: internal/pkg/wwlog/wwlog.go [hunk 1]
```diff
@@ -16,55 +16,39 @@ const (
...
+		return "[CRITICAL]"
 	}
+	return "[UNDEF]"
 }
 
 func Println(level int, message string) {
 	if level <= logLevel {
-		prefixLevel(level)
-		log.Println(message)
+		fmt.Fprintln(os.Stderr, prefixGen(level)+" "+message)
```
Comment (PaulCharlton) [inline internal/pkg/wwlog/wwlog.go:46]: @gmkurtzer are you sure you want all logs going to stderr?

File: internal/pkg/wwlog/wwlog.go [hunk 2]
```diff
@@ -16,55 +16,39 @@ const (
...
-
 	Printf(DEBUG, "Set log level to: %d\n", logLevel)
 }
 
-func SetIndent(i int) {
-	Indent = strings.Repeat(" ", i)
-}
-
-func prefixLevel(level int) {
+func prefixGen(level int) string {
```
Comment (PaulCharlton) [inline internal/pkg/wwlog/wwlog.go:27]: @gmkurtzer assuming that this gets call A LOT.    a golang "switch" on "level" would be guaranteed faster and more maintainable code over time

File: internal/pkg/wwlog/wwlog.go (hunk 1 above)
Comment (gmkurtzer) [reply internal/pkg/wwlog/wwlog.go]: Good point, fixed.

File: internal/pkg/wwlog/wwlog.go (hunk 2 above)
Comment (gmkurtzer) [reply internal/pkg/wwlog/wwlog.go:27]: Fixed, thanks for the suggestion!

**Pull request merged**